kyma.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operator.kyma-project.io/v1alpha1
```

Comment:
isn't this file generated by `hack/gen-kyma.sh` ?
I'm not sure if it needs to be versioned

Reply:
yes I will remove it